x/simulation/log.go

```diff
@@ -45,17 +47,22 @@ func (lw *StandardLogWriter) PrintLogs() {
 
 func createLogFile() *os.File {
 	var f *os.File
-	fileName := fmt.Sprintf("%s.log", time.Now().Format("2006-01-02_15:04:05"))
 
-	folderPath := os.ExpandEnv("$HOME/.simapp/simulations")
+	fileName := fmt.Sprintf("%s.log", time.Now().Format("2006-01-02_15:04:05"))
+	folderPath := path.Join(os.ExpandEnv("$HOME"), ".simapp", "simulations")
```

Comment:
why is this always simapp?

Reply:
This is used only for simulations, so it lgtm

Reply:
yeah but more apps rely on this "module", including gaia, so it would check for the .simapp directory

Reply:
Let's just use `simulation`

Reply:
This needs to be fixed in master first.